ci/release-info-v2.py

```diff
@@ -0,0 +1,96 @@
+import json
+import os
+import re
+import sys
+from datetime import date
+from subprocess import Popen, PIPE
+
+def read_version(crate):
+    # Reads package version from workspace manifest file
+    channel = Popen(
+        ['cargo', 'read-manifest', '--manifest-path', './{}/Cargo.toml'.format(crate)],
+        stdout=PIPE
+    )
+
+    # parse json data
+    data = json.load(channel.stdout)
+    version = data['version']
+    return version
+
+def check_version(ver):
+    # Checks package version for matching with the current tag reference
+    if ref is not None and ref != 'refs/tags/v' + str(ver):
+        return 0
+    else:
+        return 1
+
+def print_error(crate, ver, v):
```

Comment:
What is `v` exactly?

Reply:
v is a bit filed mapped from the consistence list to check if the version of exact package is differ from the other packages in a bundle.